File: Oddysey/UI/OddyseyUI/OddyseyUI/AudioFile.cs
using System;
using System.IO;
using System.Media;
using System.Xml.Serialization;

namespace OddyseyUI
{
    [XmlRoot("AudioFile")]
    public class AudioFile
    {
        public string Name { get; set; }
        public string Author { get; set; }
        public string Album { get; set; }
        public string Year { get; set; }
        public string Lyrics { get; set; }
        public string Length { get; set; }
        public string Score { get; set; }
        public string Data { get; set; }
        public string About { get; set; }

        public void SetMainParameters(string name, string author)
        {
            Name = name;
            Author = author;
        }

        public void GetData()
        {
            string path = @"Temp\" + Name + "-" + Author;
            Data = Convert.ToBase64String(File.ReadAllBytes(@"Temp\" + Name + "-" + Author + ".mp3"));
        }
    }
}

File: OddyseyServer/src/com/src/audio/AudioFile.java
package com.src.audio;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

//Estas son las canciones
@XmlRootElement(name = "AudioFile")
public class AudioFile {

	@XmlElement(name = "Name")
	public String name;
	@XmlElement(name = "Author")
	public String author;
	@XmlElement(name = "Album")
	public String album;
	@XmlElement(name = "Year")
	public String year;
	@XmlElement(name = "Lyrics")
	public String lyrics;
	@XmlElement(name = "Length")
	public String length;
	@XmlElement(name = "Score")
	public String score;
	@XmlElement(name = "Data")
	public String data;
	@XmlElement(name = "About")
	public String about;
	
	public AudioFile() {
		
	}
	
	public void SetMainParameters(String name, String author, String album, String score)
    {
        this.name = name;
        this.author = author;
        this.album = album;
        this.score = score;
    }
	
	//For debugging
	@Override
	public String toString() {
		return name + "/" + author + "/" + album + "/" + year + "/" + lyrics + "/" + length + "/" + score;
	}
	
	
	
	
}
